Source/GmmLib/CMakeLists.txt

```diff
@@ -146,6 +146,17 @@ MESSAGE("SourceConfiguration:")
 MESSAGE("CommonDir: ${BS_DIR_COMMON}")
 MESSAGE("IncDir: ${BS_DIR_INC}")
 
+# If '-DGMM_FIXED_MOCS_TABLE=TRUE' (default is FALSE) passed to cmake
+# configure command gmmlib will generate Fixed MOCS table depending on
+# usage requests (Gen11+). In this case on Linux user responsibility is to
+# make sure that same fixed MOCS table is programmed on KMD level.
+if (GMM_FIXED_MOCS_TABLE)
```

Comment:
IMHO, this is confusing. Why you can't reuse existing GMM_DYNAMIC_MOCS_TABLE? Besides user will get a printout in cmake log like:
```
MOCS table: Fixed
MOCS table: static
```
or wish he play with any of these settings:
```
MOCS table: Dynamic
MOCS table: static
```
which is even more confusing. Can we somehow simplify this out?

Reply:
yes, we will take a look to simplify the usage of this flag.

Reply:
Existing GMM_DYNAMIC_MOCS_TABLE to differentiate between Full MOCS table vs i915 programmed 3 entries mocs table 
New GMM_FIXED_MOCS_TABLE is to differentiate between old i915 (3 entries) vs static fixed mocs table which is different from dynamic mocs table usage.
This is why new flag was added. I can definitely change the log to simplify the usage.